### PR TITLE
Draft: [onert-micro] Bring up SparseCrossEntropy Loss

### PR DESCRIPTION
**Pull request by ljwoo94**
## What
- This commit adds feature to support `Sparse Cross Entropy` as a Loss function.
- This commit also adds `Sparse Cross Entropy Accuracy` metric to check accuracy of above loss.

## ToDo
- [ ] change test data with smaller model & data
- [ ] Revisit how to use `target_size` as I mentioned in https://github.com/Samsung/ONE/issues/13404
- [ ] Remove Sparse Cross entropy metric since it is not used for this test.

### CODE REVIEW

**Comment by ljwoo94** [comment]
TF Result: .95
Onert-micro result: .95

**Review by BalyshevArtem — APPROVED** [review]
overall lgtm, thank you!
Let's split this draft into prs and merge it

File: onert-micro/onert-micro/src/core/train/OMTrainingHandler.cpp [hunk 1]
```diff
@@ -56,11 +59,16 @@ OMStatus OMTrainingHandler::handleError(const OMConfig &config, OMRuntimeStorage
...
 
+    OMLoss loss_type = config.training_context.loss;
+
     // Get target data
     auto data_type_size = sizeof(core::OMDataType(forward_output_tensor->type()));
     size_t offset = batch_num * data_type_size * flat_size;
+    if (loss_type == SPARSE_CROSS_ENTROPY)
+    {
+        offset = batch_num * data_type_size;
+    }
```
Comment (ljwoo94) [inline onert-micro/onert-micro/src/core/train/OMTrainingHandler.cpp:70]: @BalyshevArtem 
Is it ok to changing offset according to the loss_type??
I had no idea of refactoring this code to apply sparse cross entropy... :'(
